firmware/build/extract-mrenclave

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/env bash
+
+function print_usage() {
+    echo "Usage: $0 <enclave_binary> <config_file>"
```

Comment:
I'd make clear here that there are two possible usages, one with signed enclave binaries and one with unsigned that also takes the config file. So basically two lines. It aids in the comprehension of the description that lies below.